department-of-veterans-affairs/caseflow
# Judges | Review dispositions

Connects #5505

### Description
When judges sending a finished case to dispatch, they want to review the dispositions that an attorney had set when the attorney assigned the case to them, so they can verify the issues and dispositions are correct.

### Acceptance Criteria 
- [ ] Verify that when the judge selects "Ready for dispatch", they navigate to the "Review Dispositions" screen.
- [ ] Verify that dispositions selected by their attorneys already populate in the drop downs.

Note: this page should function the same as it does for attorney check out. The only difference is the title of the page.

### Testing Plan
1. Enable `:judge_queue` and `:judge_assignment` FeatureToggles
2. Go to `/queue` as judge (`BVAAABSHIRE`), select Ready for Dispatch checkout flow
3. Confirm issue disposition dropdowns have values selected
4. If any are remanded, next page should be Review Remand Reasons, else stub for Evaluate Decision view

![screencapture-localhost-3000-queue-appeals-3617215-dispositions-2018-06-04-10_27_08](https://user-images.githubusercontent.com/8533004/40923106-e924992a-67e1-11e8-85a1-2ac072ef96a1.png)
